File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int n = nums.length;

        int total_sum = 0;
        for(int num: nums){
            total_sum += num;
        }
        
        int[] answer = new int[n];
        
        int leftsum=0;
        
        for(int i =0 ;i <n ; i++){
            int rightsum = total_sum - leftsum - nums[i];
            answer[i] = Math.abs(leftsum - rightsum);
            leftsum += nums[i];
        }
        return answer;
    }
}

File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        n = len(nums)
        total_sum = 0
        
        for num in nums:
            total_sum += num
            
        answer = [0] * len(nums)

        leftsum=0
        for i in range(len(nums)):
            rightsum = total_sum - leftsum - nums[i]
            answer[i] = abs(leftsum - rightsum)
            leftsum += nums[i]
        return answer